### ISSUE TITLE
Using Windows API to get cursor top. Not the variable `cursor_top`

### ISSUE THREAD

**Issue by CreeperMPG** [CreeperMPG/WordleCPP#1]
Use Windows API to do that.

```cpp
#include <stdio.h>
#include <Windows.h>

int main(int argc, char* argv[]) {
	HANDLE outputHandle = GetStdHandle(STD_OUTPUT_HANDLE);
	CONSOLE_SCREEN_BUFFER_INFO info;
	GetConsoleScreenBufferInfo(outputHandle, &info);
	printf("Console cursor position: x=%hd, y=%hd\n", info.dwCursorPosition.X, info.dwCursorPosition.Y);
	return 0;
}
```

**Comment by CreeperMPG** [comment]
**Will fix in next version :)**

**Comment by CreeperMPG** [comment]
**Fixed in Version 1.1**

[Release 1.1](/CreeperMPG/WordleCPP/releases/tag/1.1)
